src/files/tests/optionalChaning.spec.ts

```diff
@@ -80,12 +80,12 @@ describe('optional chaining', () => {
                 print arr?.value
                 print obj?.[0]
                 print obj?.getName()?.first?.second
-                print createObject("roByteArray")?.value
+                print createObject("roByteArray")?.Count
```

Comment:
I love it when adding more type info makes the tests fail because they were wrong :)